src/main/scala/alchemist/Message.scala

```diff
@@ -539,7 +539,7 @@ class Message() {
     System.out.println(s"$space ==============================================")
     System.out.println(s"$space Client ID:            $tempClientID")
     System.out.println(s"$space Session ID:           $tempSessionID")
-    System.out.println(s"$space Command code:         $tempCommandCode (${Commands.getName(tempCommandCode)})")
+    System.out.println(s"$space Command code:         $tempCommandCode (${Command.withValue(tempCommandCode).entryName})")
```

Comment:
since `Command` extends `UpperSnakecase` it will return a string in upper snake-case style!